### PR TITLE
Update ChangeLog

### PR DESCRIPTION
**Pull request by TheOneRing**
See https://github.com/owncloud/client/compare/v2.6.0...2.6

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+(last update on 4bbd70b796b9fee53f518dead0eed20fbf916195)
+
+Some smaller changes and bug fixes:
+* Vfs: Do not overwrite existing files by placeholder (#7557,  #7556)
+* Discovery:  Allow more HTTP error code to be treated as ignored dir (#7586)
+* GUI: Change the display name to "server (user name)", show the full text in the tooltip (#6728)
+* GUI: Limit the clickable region of the 'add folder' button (#7326)
+* GUI: Add quit button to the settings dialog (#7547)
+* GUI:  Don't show the "All files deleted" popup when unselecting everything with selective sync (#7337)
+* GUI: Don't put a too big icon in about dialog
```
Comment (HanaGemela) [inline ChangeLog]: Issue number missing

File: ChangeLog [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+* GUI: Add quit button to the settings dialog (#7547)
+* GUI:  Don't show the "All files deleted" popup when unselecting everything with selective sync (#7337)
+* GUI: Don't put a too big icon in about dialog 
+* GUI: Show a warning that proxy settings do not apply to localhost (#7169)
+* CLI: Make it possible to show settings/quit by command line invocatin (#7018, #7547)
+* Linux:  Add action to Desktop file to show settings, quit the client (#7018, #7547)
+* Shell Integration: Don't assume read-only folder when permissions are not known (#7330)
+* Sync: Temporary disable http2 support by default again (#7610)
+* Build: Port drone config to starlark
+* Build: Notify Rocket.Chat on ci results
```
Comment (HanaGemela) [inline ChangeLog]: Please remove two lines above to list only changes for the users

File: ChangeLog [hunk 3]
```diff
@@ -1,7 +1,26 @@
 ChangeLog
 =========
-
-version 2.6.0 (2019-11-)
+version 2.6.1 (2019-12-)
```
Comment (HanaGemela) [inline ChangeLog]: Add today's date

File: ChangeLog (hunk 1 above)
Comment (guruz) [reply ChangeLog]: and whitespace glitch ^^

File: ChangeLog [hunk 4]
```diff
@@ -1,7 +1,24 @@
...
+(last update on 4bbd70b796b9fee53f518dead0eed20fbf916195)
+
+Some smaller changes and bug fixes:
+* Vfs: Do not overwrite existing files by placeholder (#7557,  #7556)
+* Discovery:  Allow more HTTP error code to be treated as ignored dir (#7586)
+* GUI: Change the display name to "server (user name)", show the full text in the tooltip (#6728)
+* GUI: Limit the clickable region of the 'add folder' button (#7326)
+* GUI: Add quit button to the settings dialog (#7547)
+* GUI: Don't show the "All files deleted" popup when unselecting everything with selective sync (#7337)
+* GUI: Don't put a too big icon in about dialog (#7574)
```
Comment (HanaGemela) [inline ChangeLog]: This has not went through QA, still opened and scheduled for 2.6.2

**Comment by HanaGemela** [comment]
https://github.com/owncloud/client/issues/6884 is missing

**Pull request merged**